src/main/java/net/openhft/chronicle/network/NetworkStatsListener.java

```diff
@@ -43,15 +43,32 @@ static <N extends NetworkContext<N>> void notifyHostPort(@Nullable final Chronic
 
     void networkContext(N networkContext);
 
+    /**
+     * Called periodically with summary network stats
+     *
+     * @param writeBps write Bytes/sec
+     * @param readBps read Bytes/sec
+     * @param socketPollCountPerSecond socket poll count/sec
+     */
     void onNetworkStats(long writeBps, long readBps, long socketPollCountPerSecond);
 
     void onHostPort(String hostName, int port);
 
+    /**
+     * Notification of individual round trip latency, called if acknowledge enabled
+     *
+     * @param nanosecondLatency latency
+     */
     void onRoundTripLatency(long nanosecondLatency);
 
     default void procPrefix(String procPrefix) {
     }
 
+    /**
+     * Notification of individual message one way trip latency
+     *
```

Comment:
I think we should make it clear this value will only be accurate when the clocks of the two machines are synchronised